site/content/docs/v1.11/style-guide.md

```diff
@@ -261,7 +261,7 @@ nginx    1/1       Running   0          13s    10.200.0.4   worker0
 A list of Velero-specific terms and words to be used consistently across the site.
 
 {{< table caption="Velero.io word list" >}}
-|Term|Usage|
+|Trem|Usage|
```

Comment:
I'm not sure about the wording. To me, `Term` is short for terminology. What is the meaning of `Trem`?